### ISSUE TITLE
ActionText integration "undefined method `js_location'"

### ISSUE THREAD

**Issue by derveloper** [sferik/rails_admin#3226]
Hi,

I'm trying to use ActionText, I have setup ActionText and Webpacker according to rails documentation.
When I follow the instructions in the wiki and put the config in my rails_admin.rb initializer, I get a `undefined method `js_location' `

I've searched github for the js_location method and find nothing.

How do I get ActionText to work with rails_admin?

All ruby gems (rails, rails_admin, etc.) are on the newest versions.

Thanks in advance,
Tristan

**Comment by dennyf** [comment]
Same issue here - did you manage to resolve it?

**Comment by derveloper** [comment]
thanks, but I don't know if this was my problem, I'm not using rails anymore for my project.

**Comment by nikos83** [comment]
Hi, anyone else was having this problem? I'm having the same problem now 
undefined method `js_location' 
I've added to my model this. My model has description: string
```
has_rich_text :description

 field :description do
        js_location { bindings[:view].asset_pack_path 'actiontext.js' }
  end
```
I also have added actiontext.js to the javascript/packs/
It includes 
```
require("trix")
require("@rails/actiontext")
```
I've cleared tmp folder by:  rails tmp:clear 
But I still have this error. Before following https://github.com/railsadminteam/rails_admin/wiki/ActionText
The easy setup works properly. But I need to have support for including images in the text. Does anyone have suggestions? 
Rails 6.0.3.7
rails_admin (2.1.1)
webpacker (4.3.0)
actiontext (6.0.3.7)
